src/api/RestAPI.ts

```diff
@@ -199,7 +200,7 @@ export class RestAPI extends EventEmitter {
             });
         }
 
-        const payload: boolean = [HTTPMethod.GET, HTTPMethod.DELETE].indexOf(method) === -1;
+        const payload: boolean = [HTTPMethod.GET, HTTPMethod.DELETE].includes(method) === false;
```

Comment:
@phieronymus just a warning that `IE11` doesn't support `Array.prototype.includes`, so it needs to be polyfilled if it's not already